basyx/aas/adapter/http.py

```diff
@@ -637,15 +678,22 @@ def delete_aas_submodel_refs_specific(self, request: Request, url_args: Dict, **
         raise NotFound(f"The AAS {aas!r} doesn't have the reference {url_args['submodel_ref']!r}!")
 
     # ------ SUBMODEL REPO ROUTES -------
-    def get_submodel_all(self, request: Request, url_args: Dict, **_kwargs) -> Response:
-        response_t = get_response_type(request)
+    def _get_submodels_python(self, request: Request, url_args: Dict) -> Iterator[model.Submodel]:
```

Comment:
Yeah I wasn't really sure about the name. What do you think would be a fitting name? Since it just returns all submmodels `get_submodels` would fit but that's already taken.